### DISCUSSION TITLE
arm64 for raspberryPI debian/ubuntu?

### DISCUSSION THREAD

**Discussion by FilipStadler** [DistroAV/DistroAV#869] [Ideas]
Is it possible ?

**Comment by paulpv** [comment]
Let me revisit this.
I tried this a month or so ago ago and did get OBS + obs-ndi it to function, but it was not impressive.
Maybe things have changed.

**Reply by JuniorGamingTime** [reply]
Did this ever work? I'm trying to run OBS with NDI on a Jetson nano

**Reply by paulpv** [reply]
As I mentioned, I did get this to work on RPi4, but the performance was not impressive.
It has been 2 months and I still haven't gotten around to revisiting this. :/

**Reply by FilipStadler** [reply]
Maybe the rpi5 or simular sbc would have the  performance needed ?

**Reply by paulpv** [reply]
I have 2 RPi5s on order, but they are not scheduled to ship until January 2024.

I just contacted Raspberry .com/.org asking if they might be able to provide at least one faster, but I won't be holding my breath 

If anyone has any contacts at Raspberry then feel free to contact them.

**Reply by paulpv** [reply]
Raspberry .com/.org responded that they don't have any devices to give out, which is not true.
I guess you had to be big to get noticed.
I told them in my request that obs-ndi is open source and has thousands of users (I actually have no idea how many users were have), but I guess that wasn't enough.
Meanwhile, plenty of other big people/projects out there are happily testing on RPi5 devices that they received for free from Raspberry.

**Comment by trharmon0** [comment]
I have an 8GB Pi5 with attached Logitech Webcam C925e and am willing to do some hacking. I can put it on a semi-trusted network for a dev or I can do some hacking myself.

It is running Debian Unstable with latest packages as of today.

lmk
